rootfs/builder/install-buildpacks

```diff
@@ -29,14 +29,14 @@ download_buildpack() {
 mkdir -p $BUILDPACK_INSTALL_PATH
 
 download_buildpack https://github.com/heroku/heroku-buildpack-multi.git          26fa21a
-download_buildpack https://github.com/heroku/heroku-buildpack-ruby.git           v138
-download_buildpack https://github.com/heroku/heroku-buildpack-nodejs.git         v80
-download_buildpack https://github.com/heroku/heroku-buildpack-java.git           v39
-download_buildpack https://github.com/heroku/heroku-buildpack-gradle.git         v12
+download_buildpack https://github.com/heroku/heroku-buildpack-ruby.git           v144
+download_buildpack https://github.com/heroku/heroku-buildpack-nodejs.git         v87
+download_buildpack https://github.com/heroku/heroku-buildpack-java.git           v43
+download_buildpack https://github.com/heroku/heroku-buildpack-gradle.git         v17
 download_buildpack https://github.com/heroku/heroku-buildpack-grails.git         v19
-download_buildpack https://github.com/heroku/heroku-buildpack-play.git           v24
-download_buildpack https://github.com/heroku/heroku-buildpack-python.git         v62
-download_buildpack https://github.com/heroku/heroku-buildpack-php.git            v75
-download_buildpack https://github.com/heroku/heroku-buildpack-clojure.git        v68
-download_buildpack https://github.com/heroku/heroku-buildpack-scala.git          v60
-download_buildpack https://github.com/heroku/heroku-buildpack-go.git             v16
+download_buildpack https://github.com/heroku/heroku-buildpack-play.git           v26
+download_buildpack https://github.com/heroku/heroku-buildpack-python.git         v77
+download_buildpack https://github.com/heroku/heroku-buildpack-php.git            v92
+download_buildpack https://github.com/heroku/heroku-buildpack-clojure.git        v75
+download_buildpack https://github.com/heroku/heroku-buildpack-scala.git          v65
+download_buildpack https://github.com/heroku/heroku-buildpack-go.git             v30
```

Comment:
we should bump to v31 since that contains go1.6: https://github.com/heroku/heroku-buildpack-go/commits/v31